itsy-bitsy-data-structures.js

```diff
@@ -748,8 +748,7 @@ class Stack {
  * rather than the end. Removing the oldest items rather than the most recent.
  *
  * Again, because this limits the amount of functionality, there are many
- * different ways of implementing it. A good way might be to use a linked list
- * which we will see later.
+ * different ways of implementing it. A good way might be to use a list.
```

Comment:
Look out for the comment after the Queue implementation in which the author explains more about the current implementation and how it can be improved with the future Linked List implementation. 

```
/**
  * The important thing to note here is that because we used a list to back our
  * queue it inherits the performance of "shift" which is linear O(N) "OKAY."
  *
  * Later we'll see linked lists that will allow us to implement a much faster
  * Queue.
  */
```